### ISSUE TITLE
how do i use it?

### ISSUE THREAD

**Issue by mstrolo** [multipath-tcp/mptcp-upstream-virtme-docker#4]
can you please do some step by step?

**Comment by matttbe** [comment]
Please be more specific: is there something missing in the README file?

https://github.com/multipath-tcp/mptcp-upstream-virtme-docker#how-to-use=

Please note that this tool is useful for Kernel developers. It is not made to be used to deploy service with MPTCP.

If you are interested by using MPTCP, the easiest is to install a recent distribution (Ubuntu 22.04, Fedora 36, RedHat 9, etc.) which has MPTCP support: https://github.com/multipath-tcp/mptcp_net-next/wiki#how-to-use-mptcp=
If it is what you are looking for, please close this ticket ;-)

**Comment by sskras** [comment]
Ping @mstrolo :), it's over 3y already.

**Comment by matttbe** [comment]
> it's over 3y already.

Good point, let's close this one. It can be reopened later on if needed.
